settings/security/third-party/saml.rst

```diff
@@ -104,15 +104,45 @@ Enable SAML and enter your IdP’s details in the Admin Panel under
 .. image:: /images/settings/security/third-party/saml/zammad_connect_saml_thirdparty.png
    :alt: Example configuration of SAML
 
-.. note:: 🔏 **For the IdP certificate / certificate fingerprint:**
+Display name
+   Allows you to define a custom button name for SAML. This helps your users
+   to understand better what the button does.
```

Comment:
Replace "the button" with "the button on the login page" for failsafe clarification.